### ISSUE TITLE
Lost Bold style formatting

### ISSUE THREAD

**Issue by emnaz** [qax-os/excelize#1130]
**Description**
Lost bold style formating after update xlsx file.
Problem looks like https://github.com/qax-os/excelize/issues/420

**Steps to reproduce the issue:**
1. Create xlsx file that contain text with bold style
2. Open with excelize
3. SaveAs new file

**Describe the results you received:**
![image](https://user-images.githubusercontent.com/128556/150927486-4e1b5d12-3d93-4037-9778-5020dbc60c31.png)

**Describe the results you expected:**
![image](https://user-images.githubusercontent.com/128556/150927542-976542e6-fb2b-4158-b06c-5e77c4021168.png)

**Output of `go version`:**

```text
go version go1.17.5 windows/amd64
```

**Excelize version or commit ID:**

```text
"Path": "github.com/xuri/excelize/v2",
"Version": "v2.5.0"
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
```text
Microsoft Windows [Version 10.0.19044.1466]
Microsoft® Excel® 2019 MSO (Version 2112 Build 16.0.14729.20224) 32-bit
```

**Comment by xuri** [comment]
Thanks for your feedback, this issue similar with #1119, and has been fixed with [`236ee61`](../commit/236ee61d201e45b1fe33a58b290adb7ee32d3488), please upgrade to the master branch code.

**Comment by emnaz** [comment]
Thank you. It's work.
